### PR TITLE
Fix selecting image on thumbnail slider and vertical product thumbnails

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
- Fix bugs related to the product image component
- Fix the visualization of the component when displayed vertical in the product page.

#### What problem is this solving?
- Change main image when a thumbnail is clicked
- Change main image when new props are passed to the component
- Add padding to the thumbnail slider

#### How should this be manually tested?

[workspace](https://productimages--storecomponents.myvtex.com/ninja-300/p)

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by ClaudivanFilho** [comment]
### Bug nº 1
> quando selecionado o modo vertical do thumbnail

screenshot from 2018-07-04 15-34-58

### Bug nº 2
> quando selecionado o modo horizontal do thumbnail, não aparece a outra imagem, esse produto tem duas

screenshot from 2018-07-04 15-35-43

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- Vertical display of `ProductImage`.
+- Fix thumbnail image selection in `ProductImage`.
```
Comment (ClaudivanFilho) [inline CHANGELOG.md]: remove `Fix`

File: react/components/ProductImages/components/ThumbnailArrow.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import up from '../images/Up.svg'
+import down from '../images/Down.svg'
+import previous from '../images/Previous.svg'
+import next from '../images/Next.svg'
+
+const getUrl = (inverted, vertical) => {
```
Comment (ClaudivanFilho) [inline react/components/ProductImages/components/ThumbnailArrow.js]: getUrl -> getImage

File: react/components/ProductImages/components/ThumbnailArrow.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  if (vertical) {
+    if (inverted) {
```
Comment (ClaudivanFilho) [inline react/components/ProductImages/components/ThumbnailArrow.js]: `return inverted ? down : up`

File: react/components/ProductImages/components/ThumbnailArrow.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      return down
+    }
+
+    return up
+  }
+
+  if (inverted) {
```
Comment (ClaudivanFilho) [inline react/components/ProductImages/components/ThumbnailArrow.js]: `return inverted ? next : previous`

File: react/components/ProductImages/global.css [hunk 5]
```diff
@@ -5,133 +5,142 @@
...
 .vtex-product-image .vtex-slider__dots li button {
   color: inherit;
 }
-
 .vtex-product-image .vtex-slider__dots li button:before {
   font-size: 1rem;
   color: inherit;
 }
-
 :global .vtex-product-image .vtex-slider__dots li.slick-active button:before {
```
Comment (ClaudivanFilho) [inline react/components/ProductImages/global.css]: I think that this  `:global`  is not necessary, try to remove it

File: CHANGELOG.md (hunk 1 above)
Comment (salesfelipe) [reply CHANGELOG.md]: 👍

File: react/components/ProductImages/components/ThumbnailArrow.js (hunk 2 above)
Comment (salesfelipe) [reply react/components/ProductImages/components/ThumbnailArrow.js]: 👍

File: react/components/ProductImages/components/ThumbnailArrow.js (hunk 3 above)
Comment (salesfelipe) [reply react/components/ProductImages/components/ThumbnailArrow.js]: 👍

File: react/components/ProductImages/components/ThumbnailArrow.js (hunk 4 above)
Comment (salesfelipe) [reply react/components/ProductImages/components/ThumbnailArrow.js]: 👍

File: react/components/ProductImages/global.css (hunk 5 above)
Comment (salesfelipe) [reply react/components/ProductImages/global.css]: 👍

**Pull request merged**